### PR TITLE
Improve Avatar styles

### LINKED ISSUE #1649: Avatar component adjustments
### Description

When an avatar with a black image is placed against a pure black background, the visual boundaries of this component get blurred. 

### Figma link

_No response_

### How urgent this is?

It's blocking me right now

### PR DESCRIPTION
**Pull request by rafalpawlow**
Closes #1649 
https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-Components?node-id=104%3A76

### CODE REVIEW

**Review by eldiegod — COMMENTED** [review]
Looks good overall, would be nice to have a story with an editable avatar tho

**Review by eldiegod — APPROVED** [review]

File: src/components/Avatar/Avatar.styles.ts [hunk 1]
```diff
@@ -90,21 +85,30 @@ const getAvatarSizeCss = (size: AvatarSize): SerializedStyles => {
...
 }
 
+export const Outline = styled.div`
+  ${square('100%')};
+
+  border: 1px solid ${cVar('colorBorderAlpha')};
+  border-radius: 50%;
+  z-index: ${zIndex.overlay};
+  pointer-events: none;
+`
```
Comment (kdembler) [inline src/components/Avatar/Avatar.styles.ts]: Why do we need a separate element for that? Couldn't we just use `box-shadow` directly on the `Container`?

**Comment by kdembler** [comment]
It seems we are also not fully following the design here - the design specifies the hover/active border also for non-editable avatars. Let's update that

File: src/components/Avatar/Avatar.styles.ts [hunk 2]
```diff
@@ -113,6 +107,29 @@ export const Container = styled.div<ContainerProps>`
...
+
+  ::after {
+    ${square('100%')};
+
+    content: '';
+    display: block;
+    border-radius: 50%;
+    z-index: ${zIndex.overlay};
+    pointer-events: none;
+    box-shadow: inset 0 0 0 1px ${cVar('colorBorderAlpha')};
```
Comment (kdembler) [inline src/components/Avatar/Avatar.styles.ts]: This uses a wrong color, should be `colorBorderAlphaMuted`

**Review by kdembler — APPROVED** [review]

**Pull request merged**